src/javascript/app/pages/trade/digit_trade.js

```diff
@@ -81,42 +87,11 @@ const DigitDisplay = (() => {
             tick_count,
             {
                 quote: contract.status !== 'open' ? contract.exit_tick : spot,
-                epoch: contract.status !== 'open' ? contract.exit_tick_time : contract.date_expiry,
+                epoch: +contract.exit_tick_time || contract.current_spot_time,
```

Comment:
shouldn't we also use `+` on `current_spot_time` for consistency/assurance?

Reply:
Yes we should.